Cannot build phoenicis on ubuntu 19.04

### Expected behavior
Build phoenicis
### Actual behavior
Fails with (test):
```
[INFO] Running org.phoenicis.javafx.AppConfigurationInitialisationTest
java.util.concurrent.TimeoutException
        at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
        at org.testfx.util.WaitForAsyncUtils$ASyncFXCallable.get(WaitForAsyncUtils.java:715)
        at org.testfx.util.WaitForAsyncUtils.waitFor(WaitForAsyncUtils.java:298)
        at org.testfx.api.FxToolkit.waitForSetup(FxToolkit.java:245)
        at org.testfx.api.FxToolkit.setupFixture(FxToolkit.java:202)
        at org.phoenicis.javafx.AppConfigurationInitialisationTest.testAppConfigurationInitialisation(AppConfigurationInitialisationTest.java:49)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 32.803 s <<< FAILURE! - in org.phoenicis.javafx.AppConfigurationInitialisationTest
[ERROR] testAppConfigurationInitialisation(org.phoenicis.javafx.AppConfigurationInitialisationTest)  Time elapsed: 32.309 s  <<< FAILURE!
java.lang.AssertionError
        at org.phoenicis.javafx.AppConfigurationInitialisationTest.testAppConfigurationInitialisation(AppConfigurationInitialisationTest.java:52)
```
### System information
* software version (git revision) of phoenicis and scripts repositories https://github.com/PhoenicisOrg/phoenicis/commit/ac0e31986e720db96642c56fdbdf453bf5c78822
* used operating system ubuntu 19.04
* used Java version https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_linux-x64_bin.tar.gz

Still fails with 2000. With 20000000 I have:
Results in build failure in phoenicis-configuration:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M3:test (default-test) on project phoenicis-configuration: There was a timeout or other error in the fork -> [Help 1]
```

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java
@@ -48,8 +48,11 @@ public class JavaFXApplication extends Application {
         final MainController mainController = applicationContext.getBean(MainController.class);
         mainController.show();
         mainController.setOnClose(() -> {
-            applicationContext.getBean(ControlledThreadPoolExecutorServiceCloser.class).setCloseImmediately(true);
-            applicationContext.close();
+            try {
+                applicationContext.getBean(ControlledThreadPoolExecutorServiceCloser.class).setCloseImmediately(true);
+                applicationContext.close();
+            } catch (Exception e) {
+            }
         });
     }
 

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java
@@ -26,8 +26,11 @@ import org.phoenicis.javafx.controller.MainController;
 import org.phoenicis.multithreading.ControlledThreadPoolExecutorServiceCloser;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class JavaFXApplication extends Application {
+    private final static Logger LOGGER = LoggerFactory.getLogger(JavaFXApplication.class);
 
     public static void main(String[] args) {
         try {
@@ -52,6 +55,7 @@ public class JavaFXApplication extends Application {
                 applicationContext.getBean(ControlledThreadPoolExecutorServiceCloser.class).setCloseImmediately(true);
                 applicationContext.close();
             } catch (Exception e) {
+                LOGGER.warn("Exception while closing the application.", e);
             }
         });
     }

